socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -464,6 +469,7 @@ def test_success_with_limited_json_dump_class(self, espy_mock):
             k: a_processed_crash['json_dump'][k]
             for k in modified_config.json_dump_whitelist_keys
         }
+        del expected_processed_crash['memory_report']
```

Comment:
That is the answer to your previous question. It's how I verify that the ``memory_report`` has not been saved in the database. A few lines below there is a comparison that what has been sent to ES is the same as ``expected_processed_crash``.